lib/elasticsearch_models/base.rb

```diff
@@ -67,10 +67,14 @@ def count(**params)
 
       def distinct_values(field, additional_fields: [], where: {}, **params)
         field.presence.is_a?(String) or raise ArgumentError, "field must be a present String"
-        additional_fields.all? { |f| f.presence.is_a?(String) } or raise ArgumentError, "additional_fields must all be present Strings"
+        additional_fields.is_a?(Array) or raise ArgumentError, "additional_fields must be an Array"
+        additional_fields.all? do |f|
+          f.presence.is_a?(String) || (f.is_a?(Hash) && f.key?(:field))
+        end or raise ArgumentError, "additional_fields must all be present Strings"
```

Comment:
The error message for this exception isn't accurate, maybe `"additional_fields must all be Strings or Hashes with a :field key"`?